fuel/exceptions.py

```diff
@@ -0,0 +1,25 @@
+class AxisLabelsMismatchError(ValueError):
+    """Raised when a pair of axis labels tuples do not match."""
+    pass
+
+
+class ConfigurationError(Exception):
+    """Error raised when a configuration value is requested but not set."""
+
+
+class MissingInputFiles(Exception):
+    """Exception raised by a converter when input files are not found.
+
+    Parameters
+    ----------
```

Comment:
I know you just moved this from one file to another, but I just noticed that we're missing documentation for the `message` argument.